dask_kubernetes/tests/test_async.py

```diff
@@ -456,12 +461,12 @@ def pod_spec(image_name):
 
     start = time()
     while len(cluster.scheduler_info["workers"]) != 2:
-        await gen.sleep(0.1)
+        await asyncio.sleep(0.1)
         assert time() < start + 20
 
     a, b = list(cluster.scheduler_info["workers"])
     x = client.submit(np.ones, 1, workers=a)
-    y = client.submit(np.ones, 50_000, workers=b)
+    y = client.submit(np.ones, 50000, workers=b)
```

Comment:
This was set intentionally using the underscore notation for readability. Is there a reason why you changed it?

Reply:
ditto